### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by Uzlopak**

### CODE REVIEW

**Comment by Uzlopak** [comment]
@Aslemammad 

I need more permissions to merge changes by myself.

File: README.md [hunk 1]
```diff
@@ -1,51 +1,24 @@
-# Benchmark.js v2.1.4
+# tinybench
 
 <div align="center">
 
-[![Build Status](https://github.com/cthulhu-node/benchmark.js/workflows/ci/badge.svg)](https://github.com/cthulhu-node/benchmark.js/actions)
-[![Coverage Status](https://coveralls.io/repos/github/cthulhu-node/benchmark.js/badge.svg?branch=master)](https://coveralls.io/github/cthulhu-node/benchmark.js?branch=master)
+[![Build Status](https://github.com/tinylib/tinylib/workflows/ci/badge.svg)](https://github.com/tinylib/tinylib/actions)
```
Comment (Aslemammad) [inline README.md]: it's tinylibs and not tinylib :)

File: README.md [hunk 2]
```diff
@@ -1,51 +1,24 @@
...
 
 ```shell
-$ npm i --save benchmark
-```
-
-In Node.js:
-
-```js
-var Benchmark = require('benchmark');
+$ npm i --save tinylib
```
Comment (Aslemammad) [inline README.md]: tinybench

File: README.md [hunk 3]
```diff
@@ -57,6 +30,8 @@ npm i --save microtime
 Usage example:
 
 ```js
+var Benchmark = require('benchmark');
```
Comment (Aslemammad) [inline README.md]: tinybench :)

**Comment by Aslemammad** [comment]
Changed the access now!

File: README.md (hunk 3 above)
Comment (Uzlopak) [reply README.md]:
```suggestion
var Benchmark = require('tinybench');
```

File: README.md (hunk 2 above)
Comment (Uzlopak) [reply README.md]:
```suggestion
$ npm i --save tinybench
```

**Pull request merged**